pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidatesTest.scala

```diff
@@ -258,4 +292,45 @@ class SierraMergeCandidatesTest
         )
       )
     }
+
+  private def create962subfieldsForWellcomeImageUrl(miroId: String) =
+    create962subfieldsWith(
+      urls = List(s"http://wellcomeimages.org/indexplus/image/$miroId.html")
+    )
+
+  private def create089subfieldsWith(miroIds: List[String]): List[VarField] =
+    miroIds.map { miroId =>
+      createVarFieldWith(
+        marcTag = "089",
+        subfields = List(
+          MarcSubfield(tag = "a", content = miroId)
+        )
+      )
+    }
+
+  private def physicalAndDigitalSierraMergeCandidate(
+                                                      bibNumber: String): List[MergeCandidate] =
+    List(
+      MergeCandidate(
+        identifier = SourceIdentifier(
+          identifierType = IdentifierType("sierra-system-number"),
```

Comment:
Just a question bikeshedded from this - why do we have all these strings laying around?
Wouldn't they be safer as a type / enum?